packages/bun-types/globals.d.ts

```diff
@@ -984,7 +984,7 @@ declare interface Blob {
    * This first decodes the data from UTF-8, then parses it as JSON.
    *
    */
-  json<TJSONReturnType = any>(): Promise<TJSONReturnType>;
+  json(): Promise<unknown>;
```

Comment:
this was prior behavior but was changed to the generic because it makes every developer's experience worse.

Reply:
> this was prior behavior but was changed to the generic because it makes every developer's experience worse.

The generic was just a hidden type assertion.